<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>512</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="6,2,1">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>IP:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leIp"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>PORT:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePort"/>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pbConnect">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="pteInput"/>
      </item>
      <item row="1" column="1">
       <widget class="QListView" name="lvCom">
        <property name="editTriggers">
         <set>QAbstractItemView::EditKeyPressed</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbSend">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbAutoSend">
          <property name="text">
           <string>自动发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="pteOutput"/>
      </item>
      <item row="2" column="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>状态</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>连接：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbConnSum">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>在线：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbOnline">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>发送：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbSendSum">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>接收：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbRecvSum">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
